src/routes/PrimaryLayout/PrimaryLayout.js

```diff
@@ -252,6 +263,8 @@ const postEditorRoutes = [
   { path: `/:context(c)/:slug/${REQUIRED_NEW_POST_MATCH}` },
   { path: `/:context(c)/:slug/${REQUIRED_EDIT_POST_MATCH}` },
   { path: `/:context(c)/:slug/m/:personId/${REQUIRED_EDIT_POST_MATCH}` },
+  { path: `/:context(c)/:slug/:view(project|event|stream)/${REQUIRED_NEW_POST_MATCH}` },
```

Comment:
why is this needed now? does seem like we should talk through routing, the group stuff will also be tricky to merge with this